html/semantics/embedded-content/media-elements/playing-the-media-resource/playbackRate.html

```diff
@@ -9,11 +9,43 @@
   assert_equals(v.playbackRate, 1);
 }, 'playbackRate initial value');
 
-async_test(function(t) {
+function testPlaybackRateHelper(t, newPlaybackRate) {
   var v = document.createElement('video');
-  v.playbackRate = 2;
-  v.addEventListener('ratechange', t.step_func(function() {
+  v.addEventListener('ratechange', function() {
+    assert_equals(v.playbackRate, newPlaybackRate);
     t.done();
-  }));
-}, 'setting playbackRate');
+  });
+
+  try {
+    v.playbackRate = newPlaybackRate;
+  } catch(e) {
+    assert_equals(e.name, 'NotSupportedError');
+    assert_equals(v.playbackRate, 1 /* default */);
```

Comment:
So it looks like if a browser supports 3x, then this test will start failing, because it tests that it's equal to 1, not equal to what it was before.

I think you need to save what it was before, and compare it to that in the catch().